common/shlibs

```diff
@@ -3990,3 +3990,4 @@ libneatvnc.so.0 neatvnc-0.2.0_1
 libtdjson.so.1.6.0 libtd-1.6.0_1
 libJudy.so.1 judy-1.0.5_1
 libsignal-protocol-c.so.2 libsignal-protocol-c-2.3.3_2
+libseat.so.1 libseat-0.3.0_1
```

Comment:
If you want to avoid rebasing all the time, throw this line near another related library (perhaps libelogind?)